docs/how-to/service-catalog.md

```diff
@@ -0,0 +1,126 @@
+---
+title: Service Catalog
+description: Learn about Amplication's Service Catalog. Manage, monitor, and configure your services through an intuitive interface.
+sidebar_label: Service Catalog
+slug: /service-catalog
+---
+
+The **Service Catalog** is your central hub for managing and monitoring all services within your Amplication project.
+It provides an intuitive interface to oversee your services, manage architecture, manage changes, and configure service settings.
+
+## Overview
+
+The Service Catalog offers a comprehensive suite of tools to manage your services:
+
+- **Catalog**: View and manage your services, message brokers, and custom components in both grid and card layouts
+- [**Architecture**](/break-the-monolith/): Visualize service relationships, entities, and dependencies
+- [**Sync with Git**](/smart-git-sync/): Manage code synchronization with Git repositories
+- **Commits**: Track build history and code changes across all services
+- [**Code View**](/getting-started/view-generated-code): Access and review generated service code
+- **Pending Changes**: Monitor and commit service modifications
+- [**Settings**](/how-to/api-admin-ui-settings): Configure project-level settings for all services
+
+![Amplication Service Catalog](./assets/service-catalog/service-catalog.png)
+
+## Catalog
+
+The Catalog provides a flexible interface for viewing and managing your services, offering both grid and card layouts to suit different needs.
+You can easily filter and search through your services, and access quick actions for common management tasks.
+
+### Grid View
+
+The Grid View provides a detailed, data-rich overview of your services, displaying:
+
+- Service name and description
+- Service type and technology stack
+- Code generator version
```

Comment:
"Owner" is also one of the columns, and it's important to mention it.
There is also an option to add properties, and tags (in the workspace --> properties tab. Get to the workspace by clicking on the Amplication logo on the top left corner).
I know all of the above appear in Sandbox and not in prod, but as it's important to mention them in the docs as they are live (and soon they will be there)